iai_trajectory_generation_boxy/Trajectory[] trajectories
float32[] manipulability
string object_to_grasp
---
int32 selected_trajectory
